extensions.json

```
@@ -577,10 +577,15 @@
       "id": "giscafer.leek-fund",
       "repository": "https://github.com/giscafer/leek-fund"
     },
+    {
+      "id": "GitHub.github-vscode-theme",
+      "repository": "https://github.com/primer/github-vscode-theme",
```

It would probably be safer to checkout and pin a specific release, rather than to publish every day from the default branch:

```suggestion
      "repository": "https://github.com/primer/github-vscode-theme",
      "version": "4.1.1",
      "checkout": "v4.1.1",
```